property-services/src/main/java/org/egov/pt/web/controllers/PropertyController.java

```diff
@@ -84,8 +84,20 @@
     @PostMapping("/_search")
     public ResponseEntity<PropertyResponse> search(@Valid @RequestBody RequestInfoWrapper requestInfoWrapper,
                                                    @Valid @ModelAttribute PropertyCriteria propertyCriteria) {
-        if(!configs.getIsInboxSearchAllowed() && (ObjectUtils.isEmpty(propertyCriteria.getIsInboxSearch()) || !propertyCriteria.getIsInboxSearch()))
+        // If inbox search has been disallowed at config level, validate the search criteria.
```

Comment:
Make the Boolean field by default false. And check for only false no need for empty check